unitOneAssessmentPractice.js

```diff
@@ -5,38 +5,90 @@ let assert = require('assert')
 // Write a function called double that doubles a number
 
 // Uncomment out the next line to test your solution
+const double = (num) => {
+  return num * 2
+}
 // runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
 // Uncomment out the next line to test your solution
+const containsSeven = (number) => {
+  let num = number.toString()
+   return num.includes(7) ? true : false
```

Comment:
you can also simply write
return num.includes(7)
and that will return a boolean. No need for a ternary.